[2024-10-15T23:09:08.279+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:09:08.304+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-10-15T23:09:08.320+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:09:08.328+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-10-15T23:09:08.331+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> from DB
[2024-10-15T23:09:08.354+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-10-15T23:09:08.354+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Ready To Reschedule' PASSED: True, Task is not in reschedule mode.
[2024-10-15T23:09:08.355+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-10-15T23:09:08.355+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:09:08.360+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:09:08.366+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2024-10-15T23:09:08.366+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]>
[2024-10-15T23:09:08.367+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Ready To Reschedule' PASSED: True, Task is not in reschedule mode.
[2024-10-15T23:09:08.367+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-10-15T23:09:08.370+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:09:08.372+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:09:08.374+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2024-10-15T23:09:08.378+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-10-15T23:09:08.379+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [queued]>
[2024-10-15T23:09:08.379+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-10-15T23:09:08.392+0000] {taskinstance.py:1327} INFO - Executing <Task(ExternalTaskWithinDaysSensor): wait_for_dbt_init> on 2024-10-15 23:09:07.210288+00:00
[2024-10-15T23:09:08.396+0000] {standard_task_runner.py:57} INFO - Started process 688 to run task
[2024-10-15T23:09:08.399+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_models', 'wait_for_dbt_init', 'manual__2024-10-15T23:09:07.210288+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models.py', '--cfg-path', '/tmp/tmpenc340j8']
[2024-10-15T23:09:08.400+0000] {standard_task_runner.py:85} INFO - Job 36: Subtask wait_for_dbt_init
[2024-10-15T23:09:08.400+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7f7b0c199c10>]
[2024-10-15T23:09:08.458+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:09:08.467+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [running]> on host d867809303f2
[2024-10-15T23:09:08.467+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 688)
[2024-10-15T23:09:08.468+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 688)
[2024-10-15T23:09:08.468+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-10-15T23:09:08.470+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [running]> from DB
[2024-10-15T23:09:08.515+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:09:08.525+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-10-15T23:09:08.563+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-10-15T23:09:08.579+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='wait_for_dbt_init' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T23:09:07.210288+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T23:09:07.210288+00:00'
[2024-10-15T23:09:08.580+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-10-15T23:09:08.580+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-10-15T23:09:08.594+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2024-10-15T23:09:08.594+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-10-15T23:09:08.595+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.wait_for_dbt_init manual__2024-10-15T23:09:07.210288+00:00 [running]> from DB
[2024-10-15T23:09:08.610+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-10-15T23:09:08.610+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_models, task_id=wait_for_dbt_init, execution_date=20241015T230907, start_date=20241015T230908, end_date=20241015T230908
[2024-10-15T23:09:08.610+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 0.243231
[2024-10-15T23:09:08.625+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-10-15T23:09:08.662+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-15T23:09:08.721+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_models @ 2024-10-15 23:09:07.210288+00:00: manual__2024-10-15T23:09:07.210288+00:00, state:running, queued_at: 2024-10-15 23:09:07.224159+00:00. externally triggered: True>: 4 task(s)
[2024-10-15T23:09:08.722+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_models @ 2024-10-15 23:09:07.210288+00:00: manual__2024-10-15T23:09:07.210288+00:00, state:running, queued_at: 2024-10-15 23:09:07.224159+00:00. externally triggered: True>: 3 task(s)
[2024-10-15T23:09:08.724+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:09:08.724+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 2 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0), upstream_task_ids={'test.dbt_btc_transform.stg_kaggle__btcusdt', 'model.dbt_btc_transform.stg_kaggle__btcusdt'}
[2024-10-15T23:09:08.725+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:09:07.210288+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 2 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=0, removed=0, done=0), upstream_task_ids={'test.dbt_btc_transform.stg_kaggle__btcusdt', 'model.dbt_btc_transform.stg_kaggle__btcusdt'}
[2024-10-15T23:09:08.725+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:09:08.727+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:09:08.733+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:09:08.733+0000] {taskinstance.py:1103} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]>
[2024-10-15T23:09:08.734+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:09:08.736+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.dbt_btc_transform.stg_kaggle__btcusdt', 'wait_for_dbt_init'}
[2024-10-15T23:09:08.736+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.dbt_btc_transform.stg_kaggle__btcusdt', 'wait_for_dbt_init'}
[2024-10-15T23:09:08.737+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:09:07.210288+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:09:08.744+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
